user/applications/tinker/src/burnin_test.cpp

```diff
@@ -353,6 +418,9 @@ bool BurninTest::testSram() {
 	return test_passed;
 }
 
+const int SPI_FLASH_BUFFER_SIZE = 4096;
+static uint8_t spi_flash_buffer[SPI_FLASH_BUFFER_SIZE] = {};
```

Comment:
Should this be dynamically allocated instead?

Reply:
Nit/style: `camelCase`.

Honestly everywhere in this source as well. Let's make sure to keep things consistent: `snake_case` only in old code if rest is using `snake_case` already or in C API interfaces (implementation `camelCase`).

Reply:
Ok, i'm all over the place with style, ill change things to camelCase in a single commit to make it clear what are only formatting changes